### ISSUE TITLE
Strip request - random cable connection inside a strip

### ISSUE THREAD

**Issue by vcvrackideas** [stoermelder/vcvrack-packone#175]
Hi Ben. Yeah this sounds crazy but it might be useful for stressing test new modules and.. Tired minds) including self patching if possible. Cheers!

**Comment by stoermelder** [comment]
You mean something like this but limited to the modules in a strip?
https://library.vcvrack.com/WhatTheRack/WhatTheJack

**Comment by vcvrackideas** [comment]
Yep but maybe an option what amount of cables would be applied from one to... 20, or for all inputs (?) would be also handy tool for mind helping) sounds like cheating

**Comment by stoermelder** [comment]
I don't think I will add something like this to STRIP. Maybe a new module for this kind of weird thing. Not sure though if it is any use for someone alse...

**Comment by stoermelder** [comment]
This is a very old request, but something like this is coming with a new module

**Comment by stoermelder** [comment]
https://github.com/stoermelder/vcvrack-packone/issues/435
